src/appengine/libs/issue_management/jira/issue_tracker_manager.py

```diff
@@ -47,73 +47,94 @@ def _create_client(self):
 
   def save(self, issue):
     """Save an issue."""
-    self._update(issue)
+    if issue.id == -1:
+      return self._create(issue)
+    return self._update(issue)
 
   def create(self):
-    """Create an issue."""
-    default_fields = {
-        'project': self.project_name,
-        'summary': 'Default summary',
-        'description': 'Default description',
-        'issuetype': {
-            'name': 'Bug'
-        }
-    }
-
-    jira_issue = self.client.create_issue(fields=default_fields)
+    """Create an issue object locally."""
+    raw_fields = {"id": "-1", "fields": {"components": [], "labels": []}}
```

Comment:
nit: please use single quotes for all strings.

Reply:
done [b631f0a](https://github.com/google/clusterfuzz/pull/2797/commits/b631f0addb6aa6d0929151df53bc31a992ac6cf8)